src/backend/gdb_expansion.ts

```diff
@@ -66,7 +66,7 @@ export function expandValue(variableCreate: (arg: VariableObject | string, optio
 		stack.push(variable);
 		stack.forEach(name => {
 			prefix = "";
-			if (name != "") {
+			if (name !== "") {
```

Comment:
possible change in behavior here, would need to investigate this area more - can you leave this out this PR for now?